FitNesseRoot/HttpTestSuite/ResponseTestSuite/CreateReadUpdateDelete/content.txt

```diff
@@ -0,0 +1,26 @@
+|script    |http browser                                       |
+|set host  |localhost                                          |
+|set port  |5000                                               |
+|get       |/form/data                                         |
+|ensure    |response code equals              |404             |
+|set data  |data=fatcat                                        |
+|post      |/form                                              |
+|ensure    |response code equals              |201             |
+|ensure    |response header location equals   |/form/data      |
+|get       |/form/data                                         |
+|ensure    |response code equals              |200             |
+|ensure    |body has content                  |data=fatcat     |
+|set data  |data=heathcliff                                    |
+|put       |/form/data                                         |
+|ensure    |response code equals              |200             |
+|get       |/form/data                                         |
+|ensure    |response code equals              |200             |
+|ensure    |body has content                  |data=heathcliff |
+|delete    |/form/data                                         |
+|ensure    |response code equals              |200             |
+|get       |/form/data                                         |
+|ensure    |response code equals              |404             |
+|delete    |/form                                              |
```

Comment:
The `/form` should not be `DELETE`d. Not unless you don't expect anyone to be able to `POST` to in again. I think it makes most sense to add the `/from` directory to the public folder.